otp_app/models.py

```diff
@@ -1,3 +1,16 @@
+# models.py
 from django.db import models
 
-# Create your models here.
+class Student(models.Model):
+    enrollment_no = models.CharField(max_length=20, unique=True)
+    phone_no = models.CharField(max_length=15)
+
+    def __str__(self):
+        return self.enrollment_no
+
+class Device(models.Model):
+    enrollment_no = models.OneToOneField(Student, on_delete=models.CASCADE)
+    device_id = models.CharField(max_length=100, unique=True)
```

Comment:
Having OneToOne relation so that the student do not try to login into other mobile device without permission.